src/safeposix/filesystem.rs

```diff
@@ -321,6 +321,76 @@ pub fn convpath(cpath: &str) -> interface::RustPathBuf {
     interface::RustPathBuf::from(cpath)
 }
 
+// find your absolute pathname from inodenum
+pub fn pathnamefrominodenum(inodenum: usize) -> Option<String>{
```

Comment:
I would like a longer top level comment above the function describing this algorithm with bullet points. once we add that i think this will be good to go.